.github/workflows/test.yaml

```diff
@@ -31,8 +31,18 @@ jobs:
       - uses: actions/setup-go@v2
         with: { go-version: 1.x }
       - run: go mod download
-      - env: { KUBERNETES: "${{ matrix.kubernetes }}" }
-        run: ENVTEST_K8S_VERSION="${KUBERNETES#default}" make check-envtest
+      - run: ENVTEST_K8S_VERSION="${KUBERNETES#default}" make check-envtest
+        env:
+          KUBERNETES: "${{ matrix.kubernetes }}"
+          GO_TEST: go test --coverprofile 'envtest.coverage' --coverpkg ./internal/...
```

Comment:
Not a blocker, but I wonder if it would be useful to generate test coverage report locally to see how changes we've made might affect the coverage? If so, we could change the GO_TEST in the makefile to this, yeah?

Reply:
The `check` targets already pass the `-cover` flag which prints the percentage per package.

https://github.com/CrunchyData/postgres-operator/blob/1c3cc3597c95de3b4265a70a5982ecfc2f768ca5/Makefile#L196

```shell-session
$ make check
…
ok      github.com/crunchydata/postgres-operator/internal/config        (cached)        coverage: 96.3% of statements
…
ok      github.com/crunchydata/postgres-operator/internal/initialize    (cached)        coverage: 100.0% of statements
ok      github.com/crunchydata/postgres-operator/internal/kubeapi       (cached)        coverage: 100.0% of statements
…
```

🤔 We could generate the profile every time, but turning the profile into a meaningful report requires more tools, like `go tool cover` in this PR.

When I want to look at coverage while working on something, I usually use my editor to show it. VS Code [highlights covered lines](https://code.visualstudio.com/docs/languages/go#_test), vim-go has `:GoCoverage`, other editors probably have similar.